resources/views/users/bulk-edit.blade.php

```diff
@@ -119,6 +119,29 @@
                             </div>
                         </div> <!--/form-group-->
 
+                        <!-- activated -->
+                        <div class="form-group">
+                            <div class="col-sm-3 control-label">
+                                {{ trans('general.autoassign_licenses') }}
+                            </div>
+                            <div class="col-sm-9">
+
+                                <label for="no_change_autoassign_licenses" class="form-control">
+                                    {{ Form::radio('autoassign_licenses', '', true, ['id' => 'no_change_autoassign_licenses', 'aria-label'=>'no_change']) }}
```

Comment:
I think this `aria-label` should be `no_change_autoassign_licenses` to match the label above?
```suggestion
                                    {{ Form::radio('autoassign_licenses', '', true, ['id' => 'no_change_autoassign_licenses', 'aria-label'=>'no_change_autoassign_licenses']) }}
```